### PR TITLE
Do not error out if a service instance status cannot be determined

### PR DESCRIPTION
**Pull request by metacosm**
Mark it as `Unknown` instead.

## Was the change discussed in an issue?
No.

### CODE REVIEW

File: pkg/service/service.go [hunk 1]
```diff
@@ -264,21 +269,6 @@ func updateStatusIfMatchingDeploymentExists(dcs []appsv1.DeploymentConfig, secre
 	}
 }
 
-// GetSvcByType returns the matching (by type) service or nil of there are no matches
-func GetSvcByType(client *occlient.Client, serviceType string) (*occlient.Service, error) {
```
Comment (girishramnani) [inline pkg/service/service.go]: +1 this function is not used anywhere

**Comment by amitkrout** [comment]
@metacosm Can you please add the steps for how to test

**Comment by kadel** [comment]
Appveyor test passed, we just exceeded artifact storage quota :-(

**Comment by girishramnani** [comment]
@kadel  can we clean up the old artifacts?

**Pull request merged**